web/src/components/profile/ProfileNaicsCode.test.tsx

```diff
@@ -52,18 +55,33 @@ describe("<ProfileNaicsCode />", () => {
     expect(screen.getByText("123456")).toBeInTheDocument();
   });
 
-  it("Display Edit when there is naics code", () => {
+  it("display Edit when there is naics code", () => {
     renderComponent({ naicsCode: "624410" });
     expect(screen.getByText(configForField.editText)).toBeInTheDocument();
     expect(screen.queryByText(configForField.addText)).not.toBeInTheDocument();
   });
 
+  it("doesn't display Edit text and displays tooltip when there is a naics code and operating phase is up and running or up and running owning", () => {
+    renderComponent({
+      naicsCode: "624410",
+      operatingPhase: randomInt() % 2 ? "UP_AND_RUNNING" : "UP_AND_RUNNING_OWNING",
+    });
+    expect(screen.queryByText(configForField.editText)).not.toBeInTheDocument();
+    expect(screen.getByTestId("naics-code-tooltip")).toBeInTheDocument();
+  });
+
+  it("displays Edit text and doesn't display tooltip when there is a naics code and operating phase is not up and running or up and running owning", () => {
+    renderComponent({ naicsCode: "624410", operatingPhase: "FORMED_AND_REGISTERED" });
+    expect(screen.getByText(configForField.editText)).toBeInTheDocument();
+    expect(screen.queryByTestId("naics-code-tooltip")).not.toBeInTheDocument();
+  });
+
   it("displays Not Entered text when user has no NAICS code", () => {
     renderComponent({ naicsCode: "" });
     expect(screen.getByTestId("not-entered")).toBeInTheDocument();
   });
 
-  it("Displays Add text when naics code field is empty", () => {
+  it("displays Add text when naics code field is empty", () => {
```

Comment:
is this different between UP_AND_RUNNING_OWNING and others?

Because the Add text wouldn't link to anything for an Oscar - they don't have a roadmap

Reply:
The add or edit text would never show up for an oscar because the field only shows up if their NAICS code is populated and it also always locked. I just changed the capitalization of the word displays in the test.